tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -264,7 +401,44 @@ def test_wazuh_db_messages_global(connect_to_sockets_module, restart_wazuh, test
 
 def test_wazuh_db_chunks(restart_wazuh, configure_sockets_environment, clean_registered_agents,
                          connect_to_sockets_module, pre_insert_agents):
-    """Check that commands by chunks work properly when agents amount exceed the response maximum size"""
+    '''
+    description: Check that commands by chunks work properly when the agents' amount exceeds the response maximum size.
+                 To do this, send a command to the wazuh-db socket and check the response from the socket.
+
+    wazuh_min_version: 4.2.0
+
+    parameters:
+        - restart_wazuh:
+            type: fixture
+            brief: Reset the 'ossec.log' file and start a new monitor.
+        - configure_sockets_environment:
+            type: fixture
+            brief: Configure environment for sockets and MITM.
+        - clean_registered_agents:
+            type: fixture
+            brief: Remove all agents of wazuhdb.
+        - connect_to_sockets_module:
+            type: fixture
+            brief: Module scope version of 'connect_to_sockets' fixture.
+        - pre_insert_agents:
+            type: fixture
+            brief: Insert agents. Only used for the global queries.
+
+    assertions:
+        - Verify that the socket status response matches with 'due' to fail.
+
+    input_description:
+        - Test cases are defined in the global_messages.yaml file. Status response is expected from global
+          get-all-agents last_id 0 command, global sync-agent-info-get last_id 0 command, global
+          get-agents-by-connection-status 0 active command and global disconnect-agents 0 .* syncreq command.
```

Comment:
I would separate the cases to make them more readable.
```suggestion
        - Test cases are defined in the global_messages.yaml file. Status response is expected from 'global
          get-all-agents last_id 0', 'global sync-agent-info-get last_id 0' , 'global
          get-agents-by-connection-status 0 active' and r'global disconnect-agents 0 .* syncreq' commands.
```

Reply:
Done in the following [commit](https://github.com/wazuh/wazuh-qa/commit/5ceb7bd05d809dc92ccde0df5d314a5914036df8)